### PR TITLE
Feature/docs

### PR DESCRIPTION
**Pull request by Jammy2211**
Documentation on building a scientific workflow.

### CODE REVIEW

File: docs/overview/statistical_methods.rst [hunk 1]
```diff
@@ -8,20 +8,106 @@ Statistical Methods
 Graphical Models
 ----------------
 
+For inference problems consisting of many datasets, the model composition is often very complex. Model parameters
+can depend on multiple datasets, and the datasets themselves may be interdependent.
+
+Graphical models concisely describe these model and dataset dependencies and allow for
+them to be fitted simultaneously, with a concise API and scientific workflow that ensures scalability to big datasets.
```
Comment (VascoSch92) [inline docs/overview/statistical_methods.rst:15]:
```suggestion
Graphical models concisely describe these model and dataset dependencies, allowing them to be fitted simultaneously. This is achieved through a concise API and scientific workflow that ensures scalability to large datasets.
```

File: docs/overview/statistical_methods.rst [hunk 2]
```diff
@@ -8,20 +8,106 @@ Statistical Methods
...
+https://github.com/Jammy2211/autofit_workspace/blob/release/notebooks/features/graphical_models.ipynb
+
 Hierarchical Models
 -------------------
 
+Hierarchical models are where multiple parameters in the model are assumed to be drawn from a common distributio.
+The parameters of this parent distribution are themselves inferred from the data, which for many problems enables
+more robust and informative model fitting.
+
+Anfull description of using hierarchical models is given below:
```
Comment (VascoSch92) [inline docs/overview/statistical_methods.rst:28]:
```suggestion
A full description of using hierarchical models is given below:

```

File: docs/overview/statistical_methods.rst [hunk 3]
```diff
@@ -8,20 +8,106 @@ Statistical Methods
...
+them to be fitted simultaneously, with a concise API and scientific workflow that ensures scalability to big datasets.
+
+A full description of using graphical models is given below:
+
+https://github.com/Jammy2211/autofit_workspace/blob/release/notebooks/features/graphical_models.ipynb
+
 Hierarchical Models
 -------------------
 
+Hierarchical models are where multiple parameters in the model are assumed to be drawn from a common distributio.
```
Comment (VascoSch92) [inline docs/overview/statistical_methods.rst:24]:
```suggestion
Hierarchical models are where multiple parameters in the model are assumed to be drawn from a common distribution.
```

File: docs/overview/statistical_methods.rst [hunk 4]
```diff
@@ -8,20 +8,106 @@ Statistical Methods
...
+The parameters of this parent distribution are themselves inferred from the data, which for many problems enables
+more robust and informative model fitting.
```
Comment (VascoSch92) [inline docs/overview/statistical_methods.rst:26]:
```suggestion
The parameters of this parent distribution are themselves inferred from the data, , enabling more robust and informative model fitting for many problems.
```

File: docs/overview/statistical_methods.rst [hunk 5]
```diff
@@ -8,20 +8,106 @@ Statistical Methods
...
+to determine the most likely model parameters at any point in time.
+
+**PyAutoFit**'s interpolation feature allows for this, and a full description of its use is given below:
+
+https://github.com/Jammy2211/autofit_workspace/blob/release/notebooks/features/interpolate.ipynb
+
 Search Grid Search
 ------------------
 
+A classic method to perform model-fitting is a grid search, where the parameters of a model are divided on to a grid of
```
Comment (VascoSch92) [inline docs/overview/statistical_methods.rst:64]:
```suggestion
A classic method to perform model-fitting is a grid search, where the parameters of a model are divided onto a grid of

```

**Review by rhayes777 — APPROVED** [review]

File: docs/overview/statistical_methods.rst (hunk 5 above)
Comment (Jammy2211) [reply docs/overview/statistical_methods.rst:64]: Thanks for all the suggestions! Implemented :)

**Pull request merged**